### PR TITLE
Accession support

### PR DESCRIPTION
**Pull request by Chris7**
This creates a new model type, accession, that lets users assign a name to a subsequence of a path. There's support in here for a convention to use in vcf files for specifying that a change should be given a particular accession.

### CODE REVIEW

File: fixtures/simple.gfa [hunk 1]
```diff
@@ -1,9 +1,9 @@
 H	VN:Z:1.2
```
Comment (Chris7) [inline fixtures/simple.gfa:1]: i changed this file to be the same sequence as simple.fa so we can use the same vcfs for both.

Comment (dkhofer) [reply fixtures/simple.gfa:1]: :+1: Cool

File: migrations/core/01-initial/up.sql [hunk 2]
```diff
@@ -46,6 +46,43 @@ CREATE TABLE path (
 ) STRICT;
 CREATE UNIQUE INDEX path_uidx ON path(block_group_id, name);
 
+CREATE TABLE accession (
+  id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+  name TEXT NOT NULL,
+--  path accessions can reference other path accessions
+  path_id INTEGER NOT NULL,
+  accession_id INTEGER,
```
Comment (dkhofer) [inline migrations/core/01-initial/up.sql:54]: Is there some way to name this to make it clearer why we're referencing another accession?  parent_accession_id?  previous_accession_id?

File: src/models/accessions.rs [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+                }
+            }
+            Err(_) => {
+                panic!("something bad happened.")
+            }
+        }
+    }
+}
+
+impl Query for Accession {
```
Comment (dkhofer) [inline src/models/accessions.rs:131]: Nice

File: src/updates/vcf.rs [hunk 4]
```diff
@@ -206,51 +212,92 @@ pub fn update_with_vcf(
...
-            let sample_bg_id = BlockGroupCache::lookup(
+            let mut sample_bg_id = BlockGroupCache::lookup(
                 &mut block_group_cache,
                 collection_name,
                 &fixed_sample,
                 seq_name.clone(),
             );
-            let sample_path = PathCache::lookup(&mut path_cache, sample_bg_id, seq_name.clone());
+            if sample_bg_id.is_err() {
+                println!("can't find sample bg....check this out more");
```
Comment (dkhofer) [inline src/updates/vcf.rs:243]: Maybe this should be a panic?

**Pull request merged**